modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleFDSPropsTransformer.js

```diff
@@ -25,7 +25,14 @@ export default function propsTransformer({
 		},
 		onActionDropdownItemClick({action, itemData}) {
 			if (action.data.id === 'sampleMessage') {
-				alert(`${greeting} ${itemData.title}!`);
+				if (Liferay.__FF__.enableCustomDialogs) {
```

Comment:
(optional) It is so awkward to see a verb for a boolean variable :disappointed: Verbs should be used for functions/methods, `function enableCustomDialogs()`. Can we rename this to `customDialogsEnabled`?

Reply:
Can we do this? I can easily switch then :)

Reply:
I'd give this 80% chance to of pass Brian :smile: